src/export-page/export-stepper/ExportStepper.jsx

```diff
@@ -33,7 +34,7 @@ const ExportStepper = ({ intl, courseId }) => {
     const id = setInterval(() => {
       if (isStopFetching) {
         clearInterval(id);
-      } else {
+      } else if (savingStatus === RequestStatus.SUCCESSFUL) {
         dispatch(fetchExportStatus(courseId));
       }
```

Comment:
Should we do something by default in this condition?

Reply:
No, thats to prevent the next problem:
when a request to start exporting takes a long time to reach the backend, and the Stepper has already been enabled and started fetching - fetch can be faster than first request and after that first request return `1` as status and Step in this case return to back

Reply:
The same behaviour I repeated in import flow